schedule/kernel/ibtest-slave.yaml

```diff
@@ -33,8 +33,8 @@ schedule:
     - installation/user_settings_root
     - installation/resolve_dependency_issues
     - installation/installation_overview
-    - installation/disable_grub_timeout
```

Comment:
Just curious, this is completely replaced by new `grub_baremetal`?

Reply:
I'm not sure this was the right way to do this, but disable_grub_timeout did not work for me on the ipmi console